selenium-system-test/src/test/scala/htsstride/steps/EligibilitySteps.scala

```diff
@@ -84,4 +84,9 @@ class EligibilitySteps extends Steps with NINOGenerator {
   Then("^the account is successfully created$") {
     Browser.checkCurrentPageIs(AccountCreatedPage)
   }
+
+  And("^the SCR applicant is eligible$") {
```

Comment:
would it not make more sense to stick this in `SCRCustomerJourneySteps` as it does come under that journey only